packages/react/transform/crates/swc_plugin_text/tests/__swc_snapshots__/lib.rs/should_transform_single_static_text_to_text_attr.js

```diff
@@ -0,0 +1,8 @@
+<>
+      <text text="Hello World"></text>
+      <text text=" "></text>
+      <text></text>
```

Comment:
Does this (`<text></text>`) create a `<raw-text>`? Should we also avoid creating a `<raw-text>` for empty `<text>`? E.g. `<text></text>` -> `<text text="" />`